engine/src/engine.rs

```diff
@@ -1918,6 +1918,14 @@ impl<'env, I: IO + Copy, E: Env, M: ModExpAlgorithm> Backend for Engine<'env, I,
     fn original_storage(&self, address: H160, index: H256) -> Option<H256> {
         Some(self.storage(address, index))
     }
+
+    fn blob_gasprice(&self) -> Option<u128> {
+        Some(1)
```

Comment:
Should it be 0 instead?

Reply:
It's a good question. I decided that if: `MIN_BASE_FEE_PER_BLOB_GAS = 1` then we (_probably_) should put it to `1`.